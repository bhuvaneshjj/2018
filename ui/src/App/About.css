.About_row {
  display: flex;
  flex-direction: row;
}

.About_app {
  margin-left: 2em;
}

.About_timesync {
  display: flex;
  flex-grow: auto;
  margin-top: 2ex;
}
