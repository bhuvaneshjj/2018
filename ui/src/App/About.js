import React from 'react';
import PropTypes from 'prop-types';
import { Glyphicon, Panel, Well } from 'react-bootstrap';
import { version } from '../../package.json';
import logo from './logo.svg';
import './About.css';

const About = ({ username }) => (
  <Well>
    <Panel header="Jirkovský crossmarathon 2018" bsStyle="info">
      <div className="About_row">
        <img src={logo} className="App-logo-animated" alt="logo" />
        <div>
          Aplikace <Glyphicon glyph="star" />Jirkovský crossmarathon 2018<Glyphicon glyph="star" />
          <br />
          Verze: {version}
          <br />
          Origin: {(window && window.location && window.location.origin) || ''}
          <br />
          Přihlášen jako: {username}
        </div>
      </div>
    </Panel>
  </Well>
);

About.propTypes = {
  username: PropTypes.string
};

export default About;
