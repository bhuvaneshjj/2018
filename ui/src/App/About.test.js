import React from 'react';
import renderer from 'react-test-renderer';
import About from './About';

it('it renders', () => {
  const component = renderer.create(<About />);
  expect(component.toJSON()).toMatchSnapshot();
});
