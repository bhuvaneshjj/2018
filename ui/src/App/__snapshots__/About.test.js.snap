// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders 1`] = `
<div
  className="well"
>
  <div
    className="panel panel-info"
  >
    <div
      className="panel-heading"
    >
      Jirkovský crossmarathon 2018
    </div>
    <div
      className="panel-body"
    >
      <div
        className="About_row"
      >
        <img
          alt="logo"
          className="App-logo-animated"
          src="logo.svg"
        />
        <div>
          Aplikace 
          <span
            className="glyphicon glyphicon-star"
          />
          Jirkovský crossmarathon 2018 
          <span
            className="glyphicon glyphicon-star"
          />
          <br />
          Verze: 
          1.0.38
          <br />
          Origin: 
          http://localhost
          <br />
          <span>
            Přihlášen jako: 
          </span>
        </div>
      </div>
    </div>
  </div>
</div>
`;
